### PR TITLE
Fixing test failures due to identity-agent-sso version upgrade

### PR DESCRIPTION
**Pull request by NipuniBhagya**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1617,7 +1617,7 @@
         <identity.inbound.auth.saml.version>5.4.3</identity.inbound.auth.saml.version>
         <identity.inbound.auth.oauth.version>5.6.29</identity.inbound.auth.oauth.version>
         <identity.inbound.auth.openid.version>5.2.5</identity.inbound.auth.openid.version>
-        <identity.agent.sso.version>5.1.14</identity.agent.sso.version>
+        <identity.agent.sso.version>5.1.17</identity.agent.sso.version>
```
Comment (emswbandara) [inline pom.xml]: can we resolve conflicts in pom.xml ?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/SAMLSSOTestCase.java [hunk 2]
```diff
@@ -415,7 +415,7 @@ public void testSAMLSSOLogout() throws Exception {
                     .SAML_RESPONSE_PARAM, samlResponse);
             String resultPage = extractDataFromResponse(response);
```
Comment (nilasini) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/SAMLSSOTestCase.java]: Is the variable resultPage used anywhere?

File: modules/samples/user-mgt/sample-custome-user-store-manager/src/main/java/org/wso2/sample/user/store/manager/CustomUserStoreManager.java [hunk 3]
```diff
@@ -1,215 +0,0 @@
...
-import org.wso2.carbon.user.api.*;
-
-import java.util.Date;
-import java.util.Map;
-
-
-/**
- * Sample User Store Manager Class
- */
-public class CustomUserStoreManager implements org.wso2.carbon.user.api.UserStoreManager {
```
Comment (nilasini) [inline modules/samples/user-mgt/sample-custome-user-store-manager/src/main/java/org/wso2/sample/user/store/manager/CustomUserStoreManager.java]: Is there any specific reason to remove this classes?

**Comment by emswbandara** [comment]
This should be merged after openid is deprecated from product.

**Comment by pulasthi7** [comment]
Hi @NipuniBhagya , @emswbandara , @nilasini,

Can we merge this now? If not please get the required changes done as soon as possible.

**Comment by pulasthi7** [comment]
Closing this PR as this is sent to the 5.5.x branch. Please get the requested changes done and send the PR to the master branch instead.

**Pull request closed without merging**